README.md

```diff
@@ -79,7 +79,7 @@ To ensure the SDK is functioning correctly, we have a comprehensive test suite.
   docker compose up -d
   ```
 
-3. Run the test command. This will test the SDK against test server, configured in `.env.test`.
+4. Run the test command. This will test the SDK against test server, configured in `.env.test`.
```

Comment:
One question regarding the ```bash
   docker compose up -d
   ```

Is it syncing EigenLayer-AVS remotely, or do we need to locally git pull and get the branch ready?

Reply:
Let’s add a note here: Every commit on every branch of https://github.com/AvaProtocol/EigenLayer-AVS will auto-generate a Docker image. By default the docker compose up -d is pulling the latest commit on the main branch.

Reply:
updated.